RandomImage/app.js

```diff
@@ -0,0 +1,38 @@
+console.log("hi");
+
+
+
+// the type of the event 
+const eventType = 'click';
+
+// the target of the event 
+
+const myButton = document.querySelector('button#randomize');
+let url = ("https://source.unsplash.com/1600x900");
```

Comment:
to make the API work with with different height and width, 

you can get the values from the HTML DOM and use the variables a part of the URL

a good example 
```
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");

let url = ("https://source.unsplash.com/" + width.value + "x" + height.value);
let imageDiv = document.querySelector(".image");
image.setAttribute("src", url);
imageDiv.appendChild(image);
```